ui/app/templates/components/calendar-widget.hbs

```diff
@@ -0,0 +1,285 @@
+          >
+            January
+          </button>
+          <button
+            type="button"
+            {{on "click" (fn this.selectEndMonth 1 this.displayYear)}}
+            class="is-month-list"
+            id={{concat "1-" this.displayYear}}
+          >
+            February
+          </button>
+          <button
+            type="button"
+            {{on "click" (fn this.selectEndMonth 2 this.displayYear)}}
+            class="is-month-list"
+            id={{concat "2-" this.displayYear}}
+          >
+            March
+          </button>
+          <button
+            type="button"
+            {{on "click" (fn this.selectEndMonth 3 this.displayYear)}}
+            class="is-month-list"
+            id={{concat "3-" this.displayYear}}
+          >
+            April
+          </button>
+          <button
+            type="button"
+            {{on "click" (fn this.selectEndMonth 4 this.displayYear)}}
+            class="is-month-list"
+            id={{concat "4-" this.displayYear}}
+          >
+            May
+          </button>
+          <button
+            type="button"
+            {{on "click" (fn this.selectEndMonth 5 this.displayYear)}}
+            class="is-month-list"
+            id={{concat "5-" this.displayYear}}
+          >
+            June
+          </button>
+          <button
+            type="button"
+            {{on "click" (fn this.selectEndMonth 6 this.displayYear)}}
+            class="is-month-list"
+            id={{concat "6-" this.displayYear}}
+          >
+            July
+          </button>
+          <button
+            type="button"
+            {{on "click" (fn this.selectEndMonth 7 this.displayYear)}}
+            class="is-month-list"
+            id={{concat "7-" this.displayYear}}
+          >
+            August
+          </button>
+          <button
+            type="button"
+            {{on "click" (fn this.selectEndMonth 8 this.displayYear)}}
+            class="is-month-list"
+            id={{concat "8-" this.displayYear}}
+          >
+            September
+          </button>
+          <button
+            type="button"
+            {{on "click" (fn this.selectEndMonth 9 this.displayYear)}}
+            class="is-month-list"
+            id={{concat "9-" this.displayYear}}
+          >
+            October
+          </button>
+          <button
+            type="button"
+            {{on "click" (fn this.selectEndMonth 10 this.displayYear)}}
+            class="is-month-list"
+            id={{concat "10-" this.displayYear}}
+          >
+            November
+          </button>
+          <button
+            type="button"
+            {{on "click" (fn this.selectEndMonth 11 this.displayYear)}}
+            class="is-month-list"
+            id={{concat "11-" this.displayYear}}
+          >
+            December
+          </button>
+        </div>
+      </div>
+    {{/if}}
+
+    {{#if this.showSingleMonth}}
```

Comment:
This is essentially a copy of the above calendar chunk with a different instance and id names. The reason I did this, even though it's a copy is because the widget needs to know what calendar we're looking at. They behave differently even if they look identical.